src/CalculatorUITestFramework/GlobalCalculatorUI.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using OpenQA.Selenium.Appium.Windows;
+using System.Drawing;
+
+namespace CalculatorUITestFramework
+{
+    /// <summary>
+    /// This class contains the UI automation objects and helper methods available when the Calculator is in any mode.
+    /// </summary>
+    public class GlobalCalculatorUI
```

Comment:
You can call this something like "CalculatorApplication"

Reply:
Renamed to "CalculatorApp."  Change will be in my next commit.